### ISSUE TITLE
Current gerbers don't have default dimensions

### ISSUE THREAD

**Issue by Wanja-L** [peej/lumberjack-keyboard#7]
When uploading the zipped PCB gerbers from latest master to JLCPCB the dimensions aren't automatically filled in:

<img width="566" alt="image" src="https://user-images.githubusercontent.com/28242442/169242735-e1919410-d7f0-4b8d-a9aa-869bf196dcf6.png">

If I switch to tag 1.6 and upload those gerbers it fills them in:

<img width="447" alt="Screen Shot 2022-05-19 at 7 42 16 PM" src="https://user-images.githubusercontent.com/28242442/169242666-0590defd-6e9c-45c2-bc75-bc9455f4ffbb.png">

Not sure if the dimensions stayed the same between the versions so I figured I'd raise it here.

I'd open a PR but I have no idea how to use KiCad. I'll download it and see if I can figure it out but if you beat me to it I wouldn't be mad either :)

**Comment by peej** [comment]
This appears to be an issue with the JLCPCB gerber loader, if the gerbers take too long to process for whatever reason then it cannot determine the size of the PCB.

I've tried exporting the gerbers with different settings but nothing I've tried appears to "fix" the issue.

Gerbers themselves only contain the shapes, JLCPCB determines the PCB size from the shapes in the outline layer within the gerber.

Perhaps an update to the documentation specifying this and the PCB dimensions to use would be the best solution until JLCPCB fix their form.
